public/js/common.js

```diff
@@ -25,6 +25,13 @@ const snack = Object.freeze({
 
 const loadingAnimation = '<div class="progress"><div class="indeterminate"></div></div>';
 
+const leaderboardTypes = Object.freeze({
+    OVERALL  : {name: 'overall', displayName: 'Overall'},
+    POINTS   : {name: 'points', displayName: 'Points'},
+    ACCURACY : {name: 'accuracy', displayName: 'Accuracy'},
+    ATTEMPT  : {name: 'attempt', displayName: 'Points Per Attempt'}
```

Comment:
Use a different name plz, this name is different than what its used for

Reply:
Fixed